.github/workflows/sync_labels.yml

```diff
@@ -0,0 +1,42 @@
+# This workflow synchronizes groups of labels that correspond
+# to items of selection list in Trac. It controls that in each
+# such case there is just one label of the list present.
+# Furthermore in the case of the state it checks the labels
+# to coincide with the corresponding review state.
+
+name: Synchronize selection list lables
+
+on:
+  pull_request_review:
+    types: [submitted]
+  issues:
+    types: [opened, reopened, closed, labeled, unlabeled]
+  pull_request:
+    types: [opened, reopened, closed, ready_for_review, converted_to_draft, synchronize, labeled, unlabeled]
+  pull_request_target:
+    types: [opened, reopened, closed, ready_for_review, converted_to_draft, synchronize, labeled, unlabeled]
+
+jobs:
+  synchronize:
+    runs-on: ubuntu-latest
+    steps:
+      # Download the Python script
+      - name: Download script
```

Comment:
This fails currently. I would say you can simply replace it now by the checkout action.

Reply:
> This fails currently. 

Yes, that's the point I made in the *Notes on Testing* section of the ticket description.

> I would say you can simply replace it now by the checkout action.

Perhaps! But wouldn't that be an unnecessary waste of resources? It would clone the entire Sage repository just to use a small file, wouldn't it? I think the download will work once the PR is merged as it is now working in my fork.

Reply:
Good point, the default checkout action indeed takes about 7secs currently. But https://github.com/marketplace/actions/checkout-files allows to checkout only certain files, and has a handy fallback for the default branch of a repo.

Reply:
Thanks for your hint. I replaced the `curl` statement by this action. It worked fine inside my fork (with good performance). But unfortunately it didn't in the `sagemath` repo:

```
Run Bhacaz/checkout-files@v2
/home/runner/work/_actions/Bhacaz/checkout-files/v2/node_modules/@octokit/request/dist-node/index.js:86
      const error = new requestError.RequestError(toErrorMessage(data), status, {
                    ^

RequestError [HttpError]: Not Found
    at /home/runner/work/_actions/Bhacaz/checkout-files/v2/node_modules/@octokit/request/dist-node/index.js:86:21
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {
  status: 404,
  response: {
    url: 'https://api.github.com/repos/sagemath/sage/contents/.github%2Fsync_labels.py',
    status: 404,
    headers: {
      'access-control-allow-origin': '*',
```

It looks like the same problem as before.

Reply:
Mhh, yeah it didn't help much. But I guess there is also no real point in making the check pass in this PR, the important point is that it is working in your fork - where it does.